src/libs/actions/User.js

```diff
@@ -168,6 +174,13 @@ function validateLogin(accountID, validateCode) {
     });
 }
 
+function isBlockedFromConcierge(expiresAt) {
+    const now = moment().format('YYYY-MM-DD');
+    const expires = moment(expiresAt).format('YYYY-MM-DD');
```

Comment:
Just an observation but it's kinda off we only block based on the day, and not the time. Looking at the code in Web-E where we block users, if we block them 2 days or a week but only use the day, effectively they'll be blocked less than that.